pkgs/top-level/all-packages.nix

```diff
@@ -4776,6 +4776,10 @@ with pkgs;
 
   slurp = callPackage ../tools/wayland/slurp { };
 
+  sway-unwrapped = callPackage ../by-name/sw/sway-unwrapped/package.nix {
```

Comment:
Not sure if mixing the old approach and `pkgs/by-name` is a good idea.

Reply:
there are already 28 by-name packages where some parameters are overridden in all-packages.nix, so there seems to be precedent? do you know of any particular policy/problem discouraging it?

Reply:
The policy is actually to do it. https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults